src/Commands/ElasticsearchIndexer.php

```diff
@@ -47,11 +47,10 @@ public function indexItem(object $item, callable|array|null $dataFilter, callabl
             return;
         }
 
-        $arrItem = $item instanceof Arrayable ? $item->toArray() : (array) $item;
         $currentValues = match (true) {
             is_callable($dataFilter) => $dataFilter($item),
-            is_null($dataFilter)     => $arrItem,
-            default                  => Arr::only($arrItem, $dataFilter),
+            is_null($dataFilter)     => $this->makeArray($item),
```

Comment:
Do we need this in the first place? 
The job already ensures the values is an array https://github.com/rapidez/core/blob/c5724460e7ae83b7b5d6f18cc47766ff4ea1fade/src/Jobs/IndexJob.php#L21